blocks/product-gallery-block/_index.scss

```diff
@@ -0,0 +1,14 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-product-gallery {
+	&--featured-image-enabled {
+		.c-carousel__slide:first-child {
```

Comment:
We shouldn't do this - blocks shouldn't know about components in this way. We should explore another option to achieve what we need here